a11y-meta-display-guide/2.0/draft/techniques/epub-metadata/index.html

```diff
@@ -627,23 +621,24 @@ <h4 id="navigation-instructions">Instructions</h4>
 						<span><b>IF</b> <var>table_of_contents_navigation</var> <b>OR</b> <var>index_navigation</var> <b>OR</b> <var>page_navigation</var> <b>OR</b> <var>next_previous_structural_navigation</var>:</span>
 
 						<ol class="condition">
-							<li>
```

Comment:
These changes don't look right. You've arranged the output by the variable names instead of the output strings. So, "index" is output before "headings" which is output before "go to page".

This also concerns me because it prioritizes the alphabetizing of the English strings. I think for output we should probably weight what we expect users want to find out than worry about alphabetizing the list. Otherwise, localization isn't just outputting a different string but also requires reordering the output instructions.

Reply:
I will make some tweaks to the display sections to put this more in order of importance or group like items together.
I still feel having the variables defined in alphabetical order is a good idea, but agree that the output should be organized potentially differently.

Reply:
Ya, I don't have any issue with the variable explanations and setup being rearranged.

Changing the output is just a bit problematic. I don't think the amount of information is so great it will present problems to read through. It's the alphabetizing precedent and what that means for presenting translations that worries me.